tracer/build/_build/BuildVariables.cs

```diff
@@ -1,16 +1,22 @@
 using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
 using Nuke.Common;
 using Nuke.Common.IO;
 
 partial class Build
 {
-    public void AddDebuggerEnvironmentVariables(Dictionary<string, string> envVars)
+    public void AddDebuggerEnvironmentVariables(Dictionary<string, string> envVars, ExplorationTestName explorationTestName, AbsolutePath explorationTestsDirectory)
```

Comment:
Please remove the two unused arguments

Reply:
Done in 5548bca4e8e8fa8f356a8f3aad74707152abccad.